redbot/core/commands/requires.py

```diff
@@ -126,16 +126,14 @@ class PrivilegeLevel(enum.IntEnum):
         # The following is simply an optimised way to check if the user has the
         # admin or mod role.
         guild_settings = ctx.bot.db.guild(ctx.guild)
-        admin_role_id = await guild_settings.admin_role()
-        mod_role_id = await guild_settings.mod_role()
-        is_mod = False
-        for role in ctx.author.roles:
-            if role.id == admin_role_id:
+
+        snowflakes = ctx.author._roles  # DEP-WARN
```

Comment:
I don't mean to nitpick too much here, but seeing it side by side with how you named the other instance of this, the other form is much more legible.

```suggestion
        member_snowflakes = ctx.author._roles  # DEP-WARN
```